src/pages/Contributing/GitConventions/index.md

```diff
@@ -1,4 +1,113 @@
 ---
 title: 'Git conventions'
-order: 10
----
\ No newline at end of file
+order: 5
+---
+
+## Create and name a new branch
+
+Each branch name consists of a `type`, an optionnal issue `id` and a `description` (_Definitions are available further down the page_)
+
+```shell
+git checkout -b type-id-description
+```
+
+> MOZAIC Team use **JIRA as their project management tool** <br> BU's use **Github issues to manage their tasks into MOZAIC**
+
+
+---
+
+
+## Create and write a commit message
+
+Each commit message consists of a `type`, an optionnal `scope` and a `description` (_Definitions are available further down the page_)
+
+```shell
+<type>(<scope>): <description>
+
+# example
+feat(component/button): add a warning color theme
+```
+
+> Commit message's lines **cannot be longer 140 characters**! <br> This leads to **more readable messages** that are easy to follow when looking through the project history. 
+
+We are **automating the releases version numbers (SEMVER) and the generation of the changelogs files by parsing the commit messages**. 
+
+It is why it is very important to follow the conventional commit conventions, and to be very thoughtful about the description you provide
+
+
+---
+
+
+## Create a new pull request
+
+The pull request naming follow the same rules as the commit message and consist of a `type`, an optionnal `scope` and a `description` (_Definitions are available further down the page_)
+
+### Manage the commit(s)
+
+It is preferable to have only one commit by pull request, but pull requests commits will be squashed before being merged to the master.
+
+The one merged commit will then be used in changelogs and releases informations .
+
+> You can manage your commits using 2 methods : <br>
+**- Rebase your commit using the rebase interactive** <br>
+**- amend your commit before push to keep the first one** <br>
+See details in [Git documentation](https://git-scm.com/book/en/v2).
+
+#### Exception
+
+If for some reason **you worked on multiple types, scopes, or tasks in the same pull request** and conserving multiple commits to be documented in the changelogs and release informations seems the best options.
+
+```shell
+feat(tools/postcss): add a plugin that does x y
+feat(pattern/buttons): create a feature that do do x y 
+```
+
+In that case the feature in the button require the feature on the dev tools, and both informations should be kept in the history, and for information purpose in the changelogs and release infos. 
+
+**You should specifically inform the reviewer that both commit should be kept when merging into master.**
+
+### WIP Status
+
+To indicate that your pull request is still Work In Progress, you can add `[WIP]` in front of your PR name and using the associated Github label.
+
+> **Warning :** WIP pull request will not be reviewed unless you specifically ask someone to
+
+
+---
+
+
+## Naming convention
```

Comment:
I think we should put this at the top, merged with the commit part, instead that telling user to read the bottom page. We could say for PR titles that they should be the same as the commit, and that's it. and we should add a link to conventional commit docs. 


We made a small workshop with the team and came up with the following conventions : 

reducing the number of possible types because we think that some are more scopes than types and using the following scopes : 

## scopes :  

- `website` : everything that as to do with the mosaic website UI, like menus, tabs nav, Gatsby etc...
- `onboard` : documentation only like getting started and contributing sections
- `patterns` : anything that is related to visual deliverables : code / design / doc / tokens / icons
- `tools` : any dev tool delivered to the end user like linters, postcss configs, tokens customization scripts etc
- `chore` : anything related to CI, devtools, scripts for mozaic developers

## Types :  

- `BREAKING_CHANGE`
- `feat`
- `fix`
- `refactor`
- `style`
- `test`

(we remove build, chore, and ci types)